library/Mockery/Generator/MockConfiguration.php

```diff
@@ -467,6 +498,14 @@ protected function getAllMethods()
             $methods = array_merge($methods, $class->getMethods());
         }
 
+        foreach ($this->getTargetTraits() AS $trait) {
+            foreach ($trait->getMethods() as $method) {
+                if ($method->isAbstract()) {
```

Comment:
It doesn't need calling there and therefore you are right, we don't need it for this implementation. I put it in there thinking I would, will back it out.